### DISCUSSION TITLE
Nix Support

### DISCUSSION THREAD

**Discussion by mhkarimi1383** [darrenburns/posting#140] [Ideas]
I'm a NixOS User
So I need a nix package to install this software on NixOS

**Comment by skrobul** [comment]
This is being worked on in https://github.com/NixOS/nixpkgs/pull/353386

For now the workaround I'm using is just installing through uv/pipx

**Comment by taha-yassine** [comment]
FYI Posting is now available on nixpkgs: https://search.nixos.org/packages?channel=unstable&query=posting
